response_operations_ui/controllers/case_controller.py

```diff
@@ -26,3 +26,59 @@ def update_case_group_statuses(short_name, period, ru_ref, event):
     response = requests.post(url, json={'event': event})
     if response.status_code != 200:
         raise ApiError(response)
+
+    logger.debug('Successfully updated case group status', short_name=short_name, period=period, ru_ref=ru_ref)
+
+
+def get_available_case_group_statuses_direct(collection_exercise_id, ru_ref):
+    logger.debug('Retrieving statuses', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref)
+    url = f'{app.config["CASE_URL"]}/casegroups/transitions/{collection_exercise_id}/{ru_ref}'
+    response = requests.get(url, auth=app.config['CASE_AUTH'])
+
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        if response.status_code == 404:
+            logger.debug('No statuses found', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref)
+            return []
+        logger.error('Error retrieving statuses', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref)
```

Comment:
Looks like logger.exception is being used in all the other PRs, might be useful to have the exception info in the logged message.